skeyan/stock-portfolio
# Make portfolio

**Changes:**
- Updated cash and stock cards in the Portfolio page to update as needed
- Modified thunks to accommodate IEX API limitations
- Minor CSS changes
- Fixes #16 , Fixes #17 , Fixes #18 , Fixes #19 

**To Test:**
- Run `npm start` on the front-end and `nodemon index.js` on the back-end. 
- Logging in and going to the Portfolio page should show cards for each stock you've purchased, or a message if none have been purchased.
